### PR TITLE
GRPC-V2 : Get Consensus Status

### PR DESCRIPTION
**Pull request by tds-1**
## Purpose

Added support for GRPC V2 getConsensusInfo

## Changes

Added support for GRPC V2 getConsensusInfo

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Added support for GRPC V2 Get Account Info
 - Added support for GRPC V2 Get Account List
+- Added support for GRPC V2 Get Concensus Status/Info.
```
Comment (EmilLa1) [inline CHANGELOG.md:7]:
```suggestion
- Added support for GRPC V2 Get Concensus Status/Info
```

File: concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java [hunk 2]
```diff
@@ -130,6 +131,19 @@ public Iterator<AccountAddress> getAccountList(final BlockHashInput input) {
         return to(grpcOutput, ClientV2MapperExtensions::to);
     }
 
+    /**
+     * Retrieve the Consensus Info  thatcontains the summary of the current state
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java:135]:
```suggestion
     * Retrieve the Consensus Info that contains the summary of the current state
```

File: concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java [hunk 3]
```diff
@@ -130,6 +131,19 @@ public Iterator<AccountAddress> getAccountList(final BlockHashInput input) {
...
+     * of the chain from the perspective of the node.
+     *
+     * @param timeoutMillis Timeout for the request in Milliseconds.
+     * @return Concensus Status ({@link ConsensusStatus})
+     */
+    public ConsensusStatus getConsensusInfo(int timeoutMillis) {
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java:141]: Should one provide a timeout here? Can't a default value be used?

**Review by EmilLa1 — COMMENTED** [review]
Looks good, a few minor comments and then this is good to go. GJ

File: CHANGELOG.md (hunk 1 above)
Comment (tds-1) [reply CHANGELOG.md]: Changed

File: concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java (hunk 2 above)
Comment (tds-1) [reply concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java]: Changed

Comment (tds-1) [reply concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java]: Changed

File: concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java (hunk 3 above)
Comment (tds-1) [reply concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java]: Removed the timeout

**Review by EmilLa1 — APPROVED** [review]

**Pull request merged**